tests/swssnet_ut.cpp

```diff
@@ -0,0 +1,64 @@
+#include <arpa/inet.h>
+#include <gtest/gtest.h>
+#include <string>
+#include <iostream>
+#include "swssnet.h"
+
+using namespace std;
+using namespace swss;
+
+TEST(IpAddress, copy)
```

Comment:
better to use TEST(module_name, submodule_name) format, so it is better to use filter to test all swssnet functionalities